core/iwasm/compilation/aot_emit_numberic.c

```diff
@@ -474,56 +480,20 @@ compile_int_div(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
                 /* Build div */
                 switch (arith_op) {
                     case INT_DIV_S:
-                        if (comp_ctx->disable_llvm_intrinsics && !is_i32
-                            && aot_intrinsic_check_capability(comp_ctx,
-                                                              "i64.div_s")) {
-                            res = aot_call_llvm_intrinsic(
-                                comp_ctx, func_ctx, "i64.div_s", param_types[0],
-                                param_types, 2, left, right);
-                        }
-                        else {
-                            LLVM_BUILD_OP(SDiv, left, right, res, "div_s",
-                                          false);
-                        }
+                        LLVM_BUILD_OP_OR_INTRINSIC(SDiv, left, right, res,
+                                                   "i64.div_s", "div_s", false);
```

Comment:
The name pass to `aot_call_llvm_intrinsic` is different after using the macro, the before is:
```C
if (comp_ctx->disable_llvm_intrinsics && !is_i32
                            && aot_intrinsic_check_capability(comp_ctx,
                                                              "i64.div_s")) {
                            res = aot_call_llvm_intrinsic(
                                comp_ctx, func_ctx, "i64.div_s", param_types[0],
                                param_types, 2, left, right);
```
The latter is:
```C
if (comp_ctx->disable_llvm_intrinsics && !is_i32
                            && aot_intrinsic_check_capability(comp_ctx,
                                                              "i64.div_s")) {
                            res = aot_call_llvm_intrinsic(
                                comp_ctx, func_ctx, "div_s", param_types[0],        => here it is "div_s" when using macro, while the original code here is "i64.div_s"
                                param_types, 2, left, right);
```